index.html

```diff
@@ -8,6 +8,75 @@
     <link rel="stylesheet" href="./styles.css" />
   </head>
   <body>
+    <main>
+      <div class="container">
+        <section class="primary">
+          <h1 class="page-title">Contrast Ratio Calculator</h1>
+          <div class="row">
+            <div class="input-fields">
+              <div class="form-item">
+                <label for="foreground">Foreground color</label>
+              </div>
+              <div class="input-items">
+                <div class="swatch-one"></div>
+                <input id="foreground" name="foreground" class="form-input" type="text" value="#000000" />
+              </div>
+            </div>
+            <div class="input-fields">
+              <div class="form-item">
+                <label for="background">Background color</label>
+              </div>
+              <div class="input-items">
+                <div class="swatch-two"></div>
+                <input id="background" name="background" class="form-input" type="text" placeholder="Second color" />
+              </div>
+            </div>
+          </div>
+          <div class="result-row">
+            <div class="ratio-result">
+              <div>
+                <p>Contrast Ratio</p>
+                <p class="large">4.51</p>
```

Comment:
Since the contrast ratio will be dynamically added, we should not hardcode a value here
We should also add an id here so we can target it in JavaScript. 


```suggestion

                <p id="contrast-ratio-result" class="large"></p>
```